Subject: [PATCH] quiet scsi ioctls warnings
Date:	Mon, 27 Dec 2004 06:20:44 +0000
X-BK-ChangeSetKey: alan@lxorguk.ukuu.org.uk[torvalds]|ChangeSet|20041227062044|30998
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2192, 2004/12/26 22:20:44-08:00, alan@lxorguk.ukuu.org.uk

	[PATCH] quiet scsi ioctls warnings
	
	SCSI ioctls can ask for a lot of memory and fail. We don't need to vomit
	in the log file for this case. Again taken from the Red Hat minor
	patches applied for FC3.
	
	Original-patch: Arjan van de Ven <arjanv@redhat.com>
	Signed-off-by: Alan Cox <alan@redhat.com>
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 scsi_ioctl.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/block/scsi_ioctl.c b/drivers/block/scsi_ioctl.c
--- a/drivers/block/scsi_ioctl.c	2004-12-26 23:27:16 -08:00
+++ b/drivers/block/scsi_ioctl.c	2004-12-26 23:27:16 -08:00
@@ -356,7 +356,7 @@
 
 	bytes = max(in_len, out_len);
 	if (bytes) {
-		buffer = kmalloc(bytes, q->bounce_gfp | GFP_USER);
+		buffer = kmalloc(bytes, q->bounce_gfp | GFP_USER| __GFP_NOWARN);
 		if (!buffer)
 			return -ENOMEM;
 
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

